packages/reports/addon/components/navi-column-config.js

```diff
@@ -113,6 +98,26 @@ class NaviColumnConfig extends Component {
     return columns;
   }
 
+  /**
+   * @property {Object} lastAddedItem - the column that has been added last
+   */
+  @computed('columns.[]', 'lastAddedColumn')
+  get lastAddedItem() {
+    const { columns, lastAddedColumn } = this;
+
+    if (lastAddedColumn) {
+      for (let i = columns.length - 1; i >= 0; i--) {
```

Comment:
Can we change this to a return columns.find(...)

Reply:
if we do `.slice().reverse()` first or something